tbx/static_src/sass/components/_footer.scss

```diff
@@ -77,6 +77,15 @@
         }
     }
 
+    &__address {
+        text-align: center;
```

Comment:
Did you get Ben E or Jeth to double check the design? I wonder if it might look better left-aligned?

Reply:
I did a left-aligned version, but Lily requested it to be centre-aligned. I'll ask if they have time for a quick look.

Reply:
The layout and content have been updated in 8491e275ef2d5c86a86485ff9314c2a948014301 to align with designs from Ben E.